managed/src/main/java/com/yugabyte/yw/controllers/CustomerTaskController.java

```diff
@@ -61,16 +65,25 @@
   }
 
   private Map<UUID, List<CustomerTaskFormData>> fetchTasks(UUID customerUUID, UUID targetUUID) {
-    Query<CustomerTask> customerTaskQuery = CustomerTask.find.query().where()
-      .eq("customer_uuid", customerUUID)
-      .orderBy("create_time desc");
-
+    Query<CustomerTask> customerTaskQuery;
+    Set<CustomerTask> pendingTasks;
     if (targetUUID != null) {
-      customerTaskQuery.where().eq("target_uuid", targetUUID);
+      String selectQuery = "SELECT * FROM customer_task " +
```

Comment:
1. We can do this without usage of raw sql queries (+ .setMaxRows to finder parameters).
2. Limit should be configurable throw RuntimeConfig (and the initial setting in configuration file).
3. And anyway SQL string preparation looks very ugly.

Reply:
Also step 2 (configurable limit) allows to test this functionality without significant efforts. (Test plan should be updated)

Reply:
@SergeyPotachev https://github.com/yugabyte/yugabyte-db/pull/7477#issuecomment-792540987 The original change was to use setMaxRows but after this suggestion I have used SQL queries.

Reply:
```
  List<CustomerTask> pendingTasks = customerTaskQuery
      .setFirstRow(0)
      .setMaxRows(2000)
      .findPagedList()
      .getList();
```
Should do what you want. 

It should add limit clause.